app/models/move.rb

```diff
@@ -12,6 +12,12 @@ class Move < ApplicationRecord
     prison_transfer: 'prison_transfer'
   }
 
+  enum cancellation_reason: {
+    error: 'error',
```

Comment:
Could we change this to something like `created_in_error` or `created_by_mistake` please? Feels like it's a little more descriptive as to what the "error" relates to.

At the moment this option is to capture cases like a user creating a move that is no longer needed or if they made a mistake.

For context, the content Leigh has come up with for the form option is:

>**Made in error**
>For example, there's a mistake or the detainee no longer needs to move